src/shared/tpm2-util.c

```diff
@@ -177,6 +177,33 @@ static int tpm2_get_capability(
         return 0;
 }
 
+static int tpm2_cache_capabilities(Tpm2Context *c) {
+        TPMU_CAPABILITIES capability;
+        TPMI_YES_NO more;
+        int r;
+
+        /* Cache the PCR capabilities, which are safe to cache, as the only way they can change is
+         * TPM2_PCR_Allocate(), which changes the allocation after the next _TPM_Init(). If the TPM is
+         * reinitialized while we are using it, all our context and sessions will be invalid, so we can
+         * safely assume the TPM PCR allocation will not change while we are using it. */
+        r = tpm2_get_capability(
+                        c,
+                        TPM2_CAP_PCRS,
+                        0,
+                        1,
```

Comment:
and similar: `/* count= */`

just to make this easier to read and indicate what this param is about